static/app/components/core/input/index.stories.tsx

```diff
@@ -5,7 +5,7 @@ import JSXNode from 'sentry/components/stories/jsxNode';
 import storyBook from 'sentry/stories/storyBook';
 import {space} from 'sentry/styles/space';
 
-import {Input} from './input';
+import {Input} from './';
 
 // eslint-disable-next-line import/no-webpack-loader-syntax
 import types from '!!type-loader!sentry/components/input';
```

Comment:
I noticed vscode wont detect these. I'm thinking of reporting any empty API references in prod to sentry so we can track if anything breaks over time

Reply:
ooh I need to change those too, right?

Reply:
92d83633